ops/pebble.py

```diff
@@ -12,7 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Client for the Pebble API (HTTP over Unix socket).
+"""Low-level Pebble API.
+
+This module provides:
```

Comment:
Now that we have a dedicated Pebble documentation site, I'd suggest linking to that in here somewhere.